keps/sig-scheduling/4815-dra-partitionable-devices/README.md

```diff
@@ -484,45 +484,44 @@ type ResourceSliceMixins struct {
   // than duplicate them.
   //
   // The total number of device mixins, device capacity consumption mixins, 
-  // capacity pool mixins, basic devices, and composite devices must be
-  // less than 128.
+  // capacity pool mixins, basic devices must be less than 128.
   //
   // +optional
   // +listType=atomic
   Device []DeviceMixin
 
-  // DeviceCapacityConsumption represents a list of capacity
-  // consumption mixins, each of which contains a set of capacities
-  // that a device will consume from a capacity pool.
+  // DeviceCounterConsumption represents a list of counters
+  // consumption mixins, each of which contains a set of counters
+  // that a device will consume from a counter set.
   //
-  // This makes it possible to define a set of shared capacities that
-  // are not tied to a specific pool. The pool is inferred by context
-  // in which the DeviceCapacityConsumptionMixin is referenced from
+  // This makes it possible to define a set of shared counters that
+  // are not tied to a specific counter set. The counter set is inferred by context
```

Comment:
it's not actually inferred, it's explicitly specified. I know it wasn't part of this change, but I just noticed it. I would say something like: "The counter set is specified when the DeviceCounterConumptionMixin is reference from the device".